apis/core/lib/handlers/csv-source.ts

```diff
@@ -7,8 +7,12 @@ import { createCSVSource } from '../domain/csv-source/upload'
 import { cc } from '@cube-creator/core/namespace'
 import { deleteSource } from '../domain/csv-source/delete'
 import { update } from '../domain/csv-source/update'
-import { getPresignedLink, setPresignedLink } from '../domain/csv-source/lib/getDownloadLink'
 import { replaceFile } from '../domain/csv-source/replace'
+import { mediaObjectFromPointer } from '../../../../packages/model/CsvSource'
```

Comment:
I think this would be 

```suggestion
import { mediaObjectFromPointer } from '@cube-creator/model/CsvSource'
```